server/controllers/articles.controller.js

```diff
@@ -1,15 +1,34 @@
-const { fetchArticle } = require("../models/articles.model");
+const articles = require("../../db/data/test-data/articles");
+const {
+  fetchArticle,
+  fetchArticles,
+  fetchCommentsByArticleId,
+} = require("../models/articles.model");
 
-exports.getArticle = (req, res) => {
+exports.getArticles = (req, res, next) => {
+  return fetchArticles().then((rows) => {
+    res.status(200).send({ articles: rows });
+  });
+};
+
+exports.getArticle = (req, res, next) => {
   const { article_id } = req.params;
   return fetchArticle(article_id)
     .then((article) => {
       res.status(200).send({ articles: article });
     })
     .catch((err) => {
-      if (err.code === "22P02") {
-        return res.status(400).send({ msg: "Bad request" });
-      }
-      res.status(404).send({ msg: "article does not exist" });
+      next(err);
+    });
+};
+
+exports.getCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  return fetchCommentsByArticleId(article_id)
```

Comment:
I like the semantic names here. Nice!

You don't need to return this function since you are sending the response just below in the .then block. You have also named this file "articles.controllers.js" but you are dealing with the comments table here, consider separating them out into a new files, same with the models.